Creating new services with `0.29.1` fails

```
│ Error: Invalid count argument
│ 
│   on .terraform/modules/services.otel_container_definition/modules/assert/main.tf line 2, in data "external" "assertion":
│    2:   count   = var.condition ? 0 : 1
│ 
│ The "count" value depends on resource attributes that cannot be determined until apply, so Terraform cannot predict how many instances will be created. To
│ work around this, use the -target argument to first apply only the resources that the count depends on.
╵
make: *** [tf] Error 1
```

Workaround: apply using `0.29.0` first.

--- a/README.md
+++ b/README.md
@@ -289,11 +289,11 @@ for example.
 | Name | Source | Version |
 |------|--------|---------|
 | <a name="module_code_deploy"></a> [code\_deploy](#module\_code\_deploy) | ./modules/deployment | n/a |
-| <a name="module_container_definition"></a> [container\_definition](#module\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 0.2.0 |
+| <a name="module_container_definition"></a> [container\_definition](#module\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 1.0.2 |
 | <a name="module_ecr"></a> [ecr](#module\_ecr) | ./modules/ecr | n/a |
-| <a name="module_envoy_container_definition"></a> [envoy\_container\_definition](#module\_envoy\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 0.2.0 |
-| <a name="module_fluentbit_container_definition"></a> [fluentbit\_container\_definition](#module\_fluentbit\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 0.2.0 |
-| <a name="module_otel_container_definition"></a> [otel\_container\_definition](#module\_otel\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 0.2.0 |
+| <a name="module_envoy_container_definition"></a> [envoy\_container\_definition](#module\_envoy\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 1.0.2 |
+| <a name="module_fluentbit_container_definition"></a> [fluentbit\_container\_definition](#module\_fluentbit\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 1.0.2 |
+| <a name="module_otel_container_definition"></a> [otel\_container\_definition](#module\_otel\_container\_definition) | cloudposse/config/yaml//modules/deepmerge | 1.0.2 |
 | <a name="module_sg"></a> [sg](#module\_sg) | registry.terraform.io/terraform-aws-modules/security-group/aws | ~> 3.0 |
 
 ## Resources

--- a/container_definition.tf
+++ b/container_definition.tf
@@ -54,7 +54,7 @@ locals {
 
 module "container_definition" {
   source  = "cloudposse/config/yaml//modules/deepmerge"
-  version = "0.2.0"
+  version = "1.0.2"
 
   maps = [
     local.app_container_defaults,

--- a/envoy.tf
+++ b/envoy.tf
@@ -62,7 +62,7 @@ locals {
 
 module "envoy_container_definition" {
   source  = "cloudposse/config/yaml//modules/deepmerge"
-  version = "0.2.0"
+  version = "1.0.2"
 
   maps = [
     local.envoy_container_defaults,

--- a/fluentbit.tf
+++ b/fluentbit.tf
@@ -39,7 +39,7 @@ locals {
 
 module "fluentbit_container_definition" {
   source  = "cloudposse/config/yaml//modules/deepmerge"
-  version = "0.2.0"
+  version = "1.0.2"
 
   maps = [
     local.fluentbit_container_defaults,

--- a/otel.tf
+++ b/otel.tf
@@ -26,7 +26,7 @@ locals {
 
 module "otel_container_definition" {
   source  = "cloudposse/config/yaml//modules/deepmerge"
-  version = "0.2.0"
+  version = "1.0.2"
 
   maps = [
     local.otel_container_defaults,

